src/test/test-fileio.c

```diff
@@ -784,7 +784,7 @@ TEST(read_line2) {
         assert_se((size_t) write(fd, buffer, sizeof(buffer)) == sizeof(buffer));
 
         assert_se(lseek(fd, 0, SEEK_SET) == 0);
-        assert_se(f = fdopen(fd, "r"));
+        assert_se(f = fdopen(fd, "re"));
```

Comment:
the "e" has no effect for fdopen(). There was a commit some time ago to remove all "e" from fdopen(), we probably should not reverse course on that.

Reply:
Yeah, this would be misleading and we shouldn't do it. Updated.